include/fty_warranty.h

```diff
@@ -1,24 +1,26 @@
 /*  =========================================================================
     fty_warranty - class description
 
-    Copyright (C) 2014 - 2017 Eaton
-
-    This program is free software; you can redistribute it and/or modify
-    it under the terms of the GNU General Public License as published by
-    the Free Software Foundation; either version 2 of the License, or
-    (at your option) any later version.
-
-    This program is distributed in the hope that it will be useful,
-    but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-    GNU General Public License for more details.
-
+    Copyright (C) 2014 - 2017 Eaton                                        
```

Comment:
Old `gsl` in use, apparently. May impact other generated code nuances.
Did you compare the actual changeset before merging?

Remember that `git difftool -y` if your friend with whatever backend it uses to compare (I like `meld`).